index.js

```diff
@@ -1,12 +1,11 @@
 "use strict";
-const pattern = /-(\w|$)/g;
+var pattern = /-(\w|$)/g;
 
-const callback = (dashChar, char) => char.toUpperCase();
+var callback = (dashChar, char) => char.toUpperCase();
```

Comment:
This will need to be a regular function. With that, I'd prefer not a variable declaration.